authority/admin/api/acme.go

```diff
@@ -0,0 +1,113 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/smallstep/certificates/api"
+	"github.com/smallstep/certificates/authority/admin"
+	"go.step.sm/linkedca"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// CreateExternalAccountKeyRequest is the type for POST /admin/acme/eab requests
+type CreateExternalAccountKeyRequest struct {
+	ProvisionerName string `json:"provisioner"`
```

Comment:
I think we can just use `Provisioner` instead of `ProvisionerName` since a name is now unique for provisioners within an authority.

Reply:
Renamed in 02cd3b6b3bf763662b24bef35279b07dd000b107.